index.html

```diff
@@ -1,151 +1,247 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="../css/styles.css" type="text/css">
-</head>
-<body>
+    <link rel="stylesheet" href="./src/css/styles.css" type="text/css" />
+  </head>
+  <body>
     <header>
-        <img src="../src/assets/images/header-shapes-mobile@2x.svg" alt="background" class="bg_shape">
-        <nav>
-            <ul class="flex-nav">
-                <li class="logo">
-                    <a href="#"><img alt="logo" src="../src/assets/images/my_logo.png"></a>
-                </li>
-                <li class="menu">
-                    <a href=""><img alt="Menu" src="../src/assets/images/img_icon.png"></a>
-                </li>
-            </ul>
-        </nav>
+      <nav class="nav-bar">
+        <ul class="list-logo">
+          <li class="logo">My Logo</li>
+        </ul>
+        <ul class="list-menu">
+          <li class="menu-icon">
+            <img src="./src/assets/images/menu-hamburger-blue.svg" alt="menu icon" />
+          </li>
+          <li class="menu-item"><a href="#">Portfolio</a></li>
+          <li class="menu-item"><a href="#">About</a></li>
+          <li class="menu-item"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="title-text">
+        <div class="reg-text">
+          <h1 class="h1-text">
+            I’m TeeJay,<br />
+            Glad to see you!
+          </h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a class="connect" href="#">LET’S CONNECT</a>
+        </div>
+
+        <ul class="icons">
+          <li><img src="./src/assets/images/twitter.svg" alt="Twitter" /></li>
+          <li><img src="./src/assets/images/linkedin.svg" alt="LinkedIn" /></li>
```

Comment:
You don't have anything like `linkedin.svg`, you have `Linkedin.svg` 😄, kindly update this. you can either rename the image name or update your codebase with the correct name.

<img width="867" alt="image" src="https://user-images.githubusercontent.com/31897434/187179802-6e23178c-6bd2-4474-9171-d74e3d2e78e2.png">